lib/float/common.sail

```diff
@@ -74,4 +77,13 @@ function float_decompose(op) = {
   }
 }
 
+/* Floating point operator implementations */
+infix    2 ==_t /* The suffix _t indicates tuple */
```

Comment:
I think these are bad for readability and it would be better to just spell it out.

Reply:
Or just scoped to the test file.

Reply:
Within a test file you could do `$include <generic_equality.sail>` and use `==`, but it shouldn't be included in ordinary library code.